.travis.yml

```diff
@@ -3,16 +3,32 @@ sudo: false
 scala:
   - "2.11.12"
   - "2.12.8"
+  - "2.13.0-RC2"
 jdk:
   - oraclejdk8
   - openjdk11
```

Comment:
Build matrix does not compose nicely with stages, but that can be improved later.

Reply:
Yeah, I noticed duplicate jobs while working on #484 and raised https://github.com/akka/akka-persistence-cassandra/issues/485